src/main/kotlin/no/nav/syfo/dialogmote/api/domain/NewReferatDTO.kt

```diff
@@ -12,6 +12,8 @@ data class NewReferatDTO(
     val narmesteLederNavn: String,
     val document: List<DocumentComponentDTO>,
     val andreDeltakere: List<NewDialogmotedeltakerAnnenDTO>,
+    val behandlerDeltatt: Boolean?,
```

Comment:
Hvis denne PR'en merges inn før det Modia tilpasses - vil det da automatisk komme inn null for disse feltene?

Reply:
Ja, feltene vil ikke komme inn fra Modia så da vil vi defaulte til true der de brukes her i isdialogmote. Tror det skal bli riktig.